Node/Generator/generators/app/index.js

```diff
@@ -33,14 +32,14 @@ module.exports = class extends Generator {
       this.destinationRoot(this.destinationPath(directoryName));
     }
 
-    this.fs.copyTpl(this.templatePath('package.json'), this.destinationPath('package.json'), { botName: directoryName });
+    this.fs.copyTpl(this.templatePath('package.json'), this.destinationPath('package.json'), {botName: directoryName});
```

Comment:
many of the spacing changes seen here and elsewhere were auto generated by 'eslint --fix'